test/e2e/webdriver/README.md

```diff
@@ -0,0 +1,1228 @@
+><br>
+>To locate an element by its class using a CSS selector, prepend the class name with a dot (.) symbol.<br>
+>
+>![Screenshot displays the send transaction screen of MetaMask, highlighting how to locate the amount text box using its class.](image/classSelector.png)
+>
+>Syntax for locating by Class
+>
+>```jsx
+>await driver.findElement('.unit-input__input’);
+>```
+></details>
+> <details><summary><b>ID - CSS selector</b></summary>
+><br>
+>To locate an element by its ID using a CSS selector, prepend the ID with a hash sign (#).<br>
+>
+>![Screenshot displays the login screen of MetaMask, highlighting how to locate the password text box using its ID.](image/idSelector.png)
+>
+>Syntax for locating by ID
+>
+>```tsx
+>await driver.findElement('#password');
+>```
+></details>
+> <details><summary><b>Attribute - CSS selector</b></summary>
+><br>
+>To target an element based on its attribute using a CSS selector, use square brackets ([]) to specify the attribute name and its value.<br>
+>
+>![Screenshot displays the overview screen of MetaMask, highlighting how to locate the button ‘Buy & Sell’ using its unique attribute **data-testid and its value**.](image/attributeSelector.png)
+>
+>Syntax for locating the attribute **data-testid**
+>
+>```tsx
+>await driver.findElement('[data-testid="eth-overview-buy"]');
+>```
+></details>
+> <details><summary><b>Attribute and tag - CSS  Selector</b></summary>
+><br>
+>Tag and attribute selectors provide a powerful way to precisely target and style HTML elements based on their type and characteristics.<br>
+>
+>![Screenshot displays the onboarding - Add custom network screen of MetaMask, highlighting how to locate the input field using the tag name and attribute type text.](image/attributeTagSelector.png)
+>
+>Syntax for locating the elements of type input text.
+>
+>```tsx
+>await driver.findElements('input[type="text"]')
+>```
+></details>
+> <details><summary><b>Locate element by link text</b></summary>
+><br>
+>This type of CSS locator applies only to hyperlink texts with the anchor tags.<br>
+>
+>![Screenshot displays the contacts screen of MetaMask, highlighting how to locate the ‘Delete contact’ link using its type as Anchor(a).](image/linkTextSelector.png)
+>
+>Syntax for locating the links
+>
+>```tsx
+>await driver.findElement({ text: 'Delete contact', tag: 'a' });
+>```
+>>
+</details>
+</details>
+<details><summary><b>Locate element by XPath</b></summary>
+
+>
+>To locate an element by XPath,to navigate through elements and attributes in the HTML document.
+>
+>![Screenshot displays ‘Confirm’ button in the send transaction page](image/xpath.png)
+>
+>Syntax for locating the button element that contains text ‘Confirm’
+>
+>```tsx
+>await driver.findClickableElement({ text: 'Confirm', tag: 'button' });
+>```
+>
+>Another syntax for locating the div menu element that contains text ‘Settings’
+>
+>```tsx
+>await driver.clickElement({ text: 'Settings', tag: 'div' });
+>```
+>>
+</details>
```

Comment:
In this example, called "Locate element by XPath," you're showing how to locate by Text and Tag (I know under the hood it becomes an XPath)